addons/website/views/snippets/s_rating.xml

```diff
@@ -48,13 +48,13 @@
             </we-row>
             <we-button-group string="Size" data-apply-to=".s_rating_icons">
                 <we-button data-select-class="" title="Small" data-img="/website/static/src/img/snippets_options/size_small.svg"/>
-                <we-button data-select-class="fa-2x" title="Medium" data-img="/website/static/src/img/snippets_options/size_medium.svg"/>
-                <we-button data-select-class="fa-3x" title="Large" data-img="/website/static/src/img/snippets_options/size_large.svg"/>
+                <we-button data-select-class="fa-lg" title="Medium" data-img="/website/static/src/img/snippets_options/size_medium.svg"/>
+                <we-button data-select-class="fa-2x" title="Large" data-img="/website/static/src/img/snippets_options/size_large.svg"/>
```

Comment:
That's ok, but it would not be too much to add 4th button for the 3x that we had.

And if you really don't want it, then you can then just *hide* it (not remove it) (example of this kind of stuff can be found with `shape_img_square_opt`). The point is that if old snippets where people used fa-3x are edited, they will be able to properly remove the fa-3x class (explain that with a comment next to the code hiding the button).